apps/arena/lib/arena/game_updater.ex

```diff
@@ -39,6 +39,10 @@ defmodule Arena.GameUpdater do
     GenServer.cast(game_pid, {:select_bounty, player_id, bounty_quest_id})
   end
 
+  def toggle_zone(game_pid) do
+    GenServer.cast(game_pid, {:toggle_zone})
```

Comment:
why using a tuple?

Reply:
Changed this!
Also updated toggle zone messages.proto due to client requested changes!